Dolapo47/Auto-Mart
# create car update status route [finish #166814380]

#### What does this PR do?
- A summary of what this task is about
This PR creates a route that enables the user to update the status of a car
#### Description of Task to be completed?
- A list of the major tasks you did to complete this feature/bug/chore
-retrieve car id from the URL
filter the database using the car id and user email
update status of the car
#### How should this be manually tested?
- Outlined steps on how someone else can successfully test this feature/chore/bug to ensure it works as expected
navigate to the patch status route
fill the status input field
send the request
#### What are the relevant pivotal tracker stories?
- PT Story link - (format - `[#166814380](https://www.pivotaltracker.com/n/projects/2346179)`)
